packages/@react-spectrum/actiongroup/docs/ActionGroup.mdx

```diff
@@ -235,18 +240,30 @@ In the default case, this merges the borders of neighboring ActionButtons. In th
 quiet case, it just reduces the margin size between the buttons.
 
 ```tsx example
-<ActionGroup density="compact">
+<ActionGroup density="compact" onAction={(key) => console.log(key)}>
   <Item key="brush" aria-label="Brush"><Brush /></Item>
   <Item key="select" aria-label="Select"><Select /></Item>
   <Item key="regionSelect" aria-label="Select Region"><RegionSelect /></Item>
 </ActionGroup>
 ```
 
-
 ```tsx example
-<ActionGroup isQuiet density="compact">
+<ActionGroup isQuiet density="compact" onAction={(key) => console.log(key)}>
   <Item key="brush" aria-label="Brush"><Brush /></Item>
   <Item key="select" aria-label="Select"><Select /></Item>
   <Item key="regionSelect" aria-label="Select Region"><RegionSelect /></Item>
 </ActionGroup>
 ```
+
+### Justified
+
+```tsx example
+<Flex width="size-2000" direction="column">
+  <ActionGroup isJustified onAction={(key) => console.log(key)}>
+    <Item key="brush" aria-label="Brush"><Brush /></Item>
+    <Item key="select" aria-label="Select"><Select /></Item>
+    <Item key="regionSelect" aria-label="Select Region"><RegionSelect /></Item>
+  </ActionGroup>
+</Flex>
+```
```

Comment:
IMO a little blurb about what `Justified` means in this case would be helpful. Just a little something to let users know that it makes the action group take all available horizontal room.